api/src/dev/server/Dockerfile

```diff
@@ -22,7 +22,6 @@ RUN apk --no-cache add \
         libc6-compat \
         openssh-client \
         git \
-        make \
```

Comment:
Can you explain this and the other removed / changed lines? They seem unrelated to the purpose of this PR. If they aren't – I'm wondering what purpose they served, or what about the Rainforest change has made them unnecessary?

Or were they just cruft to begin with?

Reply:
Sure! Those were added to support the ruby version of the rainforest cli, which is deprecated. Wanted to remove them as they are now unneeded dependencies.

Reply:
Awesome – thanks for clarifying.